src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,49 @@
+UART Device Emulation Module (DEM-UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module connects to the bus of a given CPU on one side, and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550A device, but instead of transmitting information over a UART-Interface it instead passes it to a host PC via the Debug Interconnect.
+
+.. _uart_registers:
+
+16550 UART Registers
+--------------------
+The following UART registers are implemented and accessible via the bus, the address mapping is in accordance with the UART-16550(A) standard as specified in this `Datasheet <http://caro.su/msx/ocm_de1/16550.pdf>`_.
+Currently there are no FIFOs present in the hardware, therefore writing to the ``FCR`` has no effect.
+The only bit in the ``LCR`` that has any meaning is the DLAB aka ``LCR7``, all other bits hold their written value but have no meaning.
+No Modem or DMA-Mode related features, registers or interrupts are implemented.
+Writing to a register that is not implemented has no effect, reading from such a register will always return ``0x00``.
```

Comment:
Is that behavior in accordance to the spec, i.e. does the implementation behave like a spec-compliant 16550 chip and signal the fact that no FIFOs, etc. are present through the relevant description registers?

If registers always return a fixed value you can mention that in the table below, together with the meaning of the returned value. One option would be to include two columns, access type (read, write, r/w), and reset value.